### PR TITLE
christmas music

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
Added a bunch of christmas music to play at roundstart and end for christmas
#### Changelog

:cl:  
rscadd: Added a bunch of christmas music to play at roundstart and end for christmas
/:cl:

### CODE REVIEW

**Comment by boba247** [comment]
Good idea but can we wait 2-3 weeks before porting this ?

**Comment by boodaliboo** [comment]
This  isn’t a port and won’t do anything till the 22nd of December

**Comment by oklews** [comment]
SCORE

File: code/modules/holiday/halloween.dm [hunk 1]
```diff
@@ -46,7 +46,7 @@
 /obj/structure/closet/proc/set_spooky_trap()
 	if(prob(0.1))
 		trapped = INSANE_CLOWN
-		return
+		returncode/modules/holiday/halloween.dm
```
Comment (alexkar598) [inline code/modules/holiday/halloween.dm]: no.

**Comment by boodaliboo** [comment]
Whoops Alex forgot to fix it this time I’ll fix it soon

File: code/modules/holiday/holidays.dm [hunk 2]
```diff
@@ -502,7 +502,16 @@ Since Ramadan is an entire month that lasts 29.5 days on average, the start and
...
-
+	lobby_music = list(
+		"https://www.youtube.com/watch?v=v7s2VjwQSMw", // jingle bells
+		"https://www.youtube.com/watch?v=x2YlbiyiuMc", // Winter Wonderland
+		"https://www.youtube.com/watch?v=cW38y4AFGyI", // The Grinch
+		"https://www.youtube.com/watch?v=zgKazTrhXmI", // frosty the snowman
+		"https://www.youtube.com/watch?v=oyEyMjdD2uk", // twelve days of christmas
+		"https://www.youtube.com/watch?v=maYCStVzjDs",  // deer song
+		"https://www.youtube.com/watch?v=Dkq3LD-4pmM",  // jolly christmas
+		"https://www.youtube.com/watch?v=_9ePg09zV-0",  // i got a brand new dsi for christmas
```
Comment (Amelia0010) [inline code/modules/holiday/holidays.dm]: Remove this one and I'll merge it.

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: code/modules/holiday/holidays.dm (hunk 2 above)
Comment (boodaliboo) [reply code/modules/holiday/holidays.dm]: which song?  you highlighted multiple

Comment (Amelia0010) [reply code/modules/holiday/holidays.dm]: the DSI song. Also, the last entry in the list shouldn't have a comma behind it.

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**